### ISSUE TITLE
Rust Bindings

### ISSUE THREAD

**Issue by Blykam** [gnuradio/pmt#96]
I'm building a rust app that should interface with GNU Radio over zmq, but I need pmts for that.
@bastibl built a pmt crate for his futuresdr project and [has written](https://www.bastibl.net/futuresdr-1/) about the possible compatibility with GNU Radio by providing a suitable "serde" serializer. 
I didn't manage to get that to working. So it would be really cool if there was native Rust support for pmts or at least bindings.
I tried to implement it myself, but the type magic in this project is way over my C++ skill level.

**Comment by cafeclimber** [comment]
I've been thinking about this recently! Even if it just uses something like the [cxx](https://cxx.rs/) for interop, I think it would be super cool!

**Comment by jsallay** [comment]
I agree that this would be great.  I don't have any experience with rust but would be willing to accept a PR for it.
